CHANGELOG.md

```diff
@@ -5,7 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+##[Unreleased]
+
+### Added
+- Add events `meetingReconnected`, `signalingDropped` and `receivingAudioDropped` to `eventDidReceive` by publishing them as stand alone events. Currently, these events were only included in the meeting history attribute when a meeting event is published. 
+
+
+### Changed
+- Move `toLowerCasePropertyNames` inside `Utils.ts` and add test coverage.
```

Comment:
Thanks for moving this item into the **Unreleased/Changed** section, but I don't see the item under the **[2.17.0] - 2021-09-08/Changed** section. Could you check you rebased the PR correctly?
https://github.com/aws/amazon-chime-sdk-js/blame/master/CHANGELOG.md#L30

Reply:
Kunal merged his changelog md changes by mistake so asked me to remove his comment from [2.17.0] to unreleased.

Reply:
Sorry for the confusion. If you have removed it, the GitHub diff UI should show that.

![image](https://user-images.githubusercontent.com/36240708/133163399-b9821863-9d53-495b-9a72-8d59ea744c05.png)

The PR is still out of sync. Just rebase again.
> This branch is out-of-date with the base branch